addons/web/static/src/views/kanban/kanban_record_quick_create.js

```diff
@@ -56,35 +89,153 @@ export class KanbanRecordQuickCreate extends Component {
         );
 
         // Key Navigation
-        // FIXME ? Maybe it will also validate even if enter is pressed outside of the quick create machin
         useHotkey("enter", () => this.validate("add"), { bypassEditableProtection: true });
         useHotkey("escape", () => this.cancel(true));
     }
 
-    /** @param {boolean} force */
-    cancel(force) {
+    //This is needed to avoid the auto save when leave
+    beforeLeave() {}
+
+    //This is needed to avoid the auto save when unload
+    beforeUnload() {}
+
+    //This is needed to avoid writing the id on the url
+    updateURL() {}
```

Comment:
wouldn't it be easier not to extend FormController? And do the useModel here (we probably don't need much of what the FormController does)